kombu/utils/amq_manager.py

```diff
@@ -1,22 +1,34 @@
 """AMQP Management API utilities."""
 
-
 from __future__ import annotations
 
+from typing import Any
+
+from kombu import Connection
+
+DEFAULT_RABBITMQ_PORT = 15672
+
 
-def get_manager(client, hostname=None, port=None, userid=None,
-                password=None):
+def get_manager(
+    client: Connection,
+    hostname: str | None = None,
+    port: int | None = None,
+    userid: str | None = None,
+    password: str | None = None
+):
     """Get pyrabbit manager."""
     import pyrabbit
     opt = client.transport_options.get
 
-    def get(name, val, default):
-        return (val if val is not None
-                else opt('manager_%s' % name) or
-                getattr(client, name, None) or default)
+    def get(name: str, val: Any, default: Any) -> Any:
+        return (
+            val if val is not None
+            else opt(
+                'manager_%s' % name) or getattr(client, name, None) or default
+        )
```

Comment:
This PR should keep the diff as small as possible. Please remove this reformatting.